src/packages/recompose/mapPropsStream.js

```diff
@@ -0,0 +1,17 @@
+import createEagerFactory from './createEagerFactory'
+import createHelper from './createHelper'
+import componentFromStream from './componentFromStream'
+
+const mapPropsStream = transform => BaseComponent => {
+  const factory = createEagerFactory(BaseComponent)
+  return componentFromStream(ownerProps$ =>
+    Observable.create(observer => {
```

Comment:
Can't see `.create` in proposal https://github.com/zenparsing/es-observable
may be better to use `new Observable(observer =>` instead